dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ProtocolConfig.java

```diff
@@ -465,6 +468,14 @@ public void setExtension(String extension) {
         this.extension = extension;
     }
 
+    public Boolean getEnableTelnet() {

```

Comment:
in case of boolean return value, would it not be better to method naming something like 
public Boolean isTelnetEnabled() {
    return this.enableTelnet;
}

Reply:
maybe telnetEnabled is more appropriate，I have renamed enableTelnet to telnetEnabled

Reply:
Thanks.